.github/workflows/agw-coverage.yml

```diff
@@ -120,7 +120,7 @@ jobs:
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_TITLE: "C / C++ code coverage with Bazel"
-          SLACK_USERNAME: "${{ github.workflow }}"
```

Comment:
Why did you change this? Can we be sure that `github.workflow` will never contain characters that need quoting in Yaml?

Reply:
I would boldly assume that `${{ expression }}` gives a well sanitized object back. Similar to `ls` already escaping special characters.